.github/workflows/blank.yml

```diff
@@ -1,38 +1,86 @@
-# This is a basic workflow to help you get started with Actions
+name: Formatting Review
```

Comment:
Probably need a comment to the effect of "we also do these checks in the normal azure pipelines run and do not consider effects of this one 'normative'"?